traffic_portal/test/integration/PageObjects/ParametersPage.po.ts

```diff
@@ -124,4 +125,16 @@ export class ParametersPage extends BasePage {
     })
     return result;
   }
```

Comment:
Also, `basePage` variable should be `const` at
https://github.com/apache/trafficcontrol/blob/df56f95943e45e0355ac4537a67dadf8c7a9e9cf/traffic_portal/test/integration/PageObjects/ParametersPage.po.ts#L69
https://github.com/apache/trafficcontrol/blob/df56f95943e45e0355ac4537a67dadf8c7a9e9cf/traffic_portal/test/integration/PageObjects/ParametersPage.po.ts#L98
https://github.com/apache/trafficcontrol/blob/df56f95943e45e0355ac4537a67dadf8c7a9e9cf/traffic_portal/test/integration/PageObjects/ParametersPage.po.ts#L114

`name` should be `const` too since it is never reassigned at
https://github.com/apache/trafficcontrol/blob/df56f95943e45e0355ac4537a67dadf8c7a9e9cf/traffic_portal/test/integration/PageObjects/ParametersPage.po.ts#L87